cmd/api/src/api/v2/auth/oidc.go

```diff
@@ -56,3 +62,22 @@ func (s ManagementResource) CreateOIDCProvider(response http.ResponseWriter, req
 		}
 	}
 }
+
+// DeleteOIDCProvider deletes an OIDC Provider entry
+func (s ManagementResource) DeleteOIDCProvider(response http.ResponseWriter, request *http.Request) {
+	var (
+		rawOIDCProviderID = mux.Vars(request)[api.URIPathVariableOIDCProviderID]
+	)
+
+	// Convert the incoming string url param to an int64
+	if oidcProviderID, err := strconv.ParseInt(rawOIDCProviderID, 10, 64); err != nil {
+		api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusBadRequest, err.Error(), request), response)
+	} else if err = s.db.DeleteOIDCProvider(request.Context(), oidcProviderID); errors.Is(err, database.ErrNotFound) {
+		// Handle error if requested record could not be found
+		api.WriteErrorResponse(request.Context(), api.BuildErrorResponse(http.StatusNotFound, err.Error(), request), response)
+	} else if err != nil {
+		api.HandleDatabaseError(request, response, err)
+	} else {
+		api.WriteBasicResponse(request.Context(), "", http.StatusOK, response)
```

Comment:
I think we can just write the statuscode and don't need to go through the full writeBasicResponse here